docs/react/toggling-devices.mdx

```diff
@@ -4,7 +4,32 @@ sidebar_position: 4
 
 # Toggling devices
```

Comment:
WDYT about changing title of this article, to sth about managing devices? (Managing devices/Switching devices ?)